packages/iris-grid/src/format-context-menus/FormatContextMenuUtils.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import CustomFormatAction from './CustomFormatAction';
+import { TableColumnFormatter, TableColumnFormat } from '../formatters';
+
+export interface FormatContextMenuOption {
+  title: string;
+  description: string;
+  group: number;
+  format: TableColumnFormat | null;
+  isSelected: boolean;
+}
+
+export interface CustomFormatOption extends FormatContextMenuOption {
```

Comment:
Random side note - you use `interface` in many places I would've just used `type`. Any particular reason for that? The differences are pretty minor...

Edit: Re-reading the latest TypeScript docs, it says to use `interface` until you need `type`, and the only case I think that you would need `type` is renaming a primitive... so carry on :) 
> For the most part, you can choose based on personal preference, and TypeScript will tell you if it needs something to be the other kind of declaration. If you would like a heuristic, use interface until you need to use features from type.
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Reply:
Ya mostly that. Types are also needed for unions like `TypeA | TypeB`